src/h5web/vis-packs/core/heatmap/MappedHeatmapVis.tsx

```diff
@@ -56,16 +62,13 @@ function MappedHeatmapVis(props: Props): ReactElement {
   const baseArray = useBaseArray(value, slicedDims);
   const dataArray = useMappedArray(baseArray, slicedMapping);
 
-  const domain = useMemo(() => {
-    return customDomain || getDomain(dataArray.data as number[], scaleType);
-  }, [customDomain, dataArray, scaleType]);
+  const dataDomain =
+    useDomain(dataArray.data as number[], scaleType) || DEFAULT_DOMAIN;
```

Comment:
I need `|| DEFAULT_DOMAIN` in order to compute the `visDomain` below. I can't wait for `dataDomain` to be set in the config with the `useEffect`. I may eventually move the fallback into `useDomain`, if it also makes sense for the line vis.